docs/docs/tutorials/tutorials/contract_tutorials/token_contract.md

```diff
@@ -411,7 +411,7 @@ A getter function for checking the token `total_supply`.
 
 #### `balance_of_private`
 
-A getter function for checking the private balance of the provided Aztec account. Note that the [Private Execution Environment (PXE)](https://github.com/AztecProtocol/aztec-packages/tree/#include_aztec_version/yarn-project/pxe) must have access to the `owner`s decryption keys in order to decrypt their notes.
+A getter function for checking the private balance of the provided Aztec account. Note that the [Private Execution Environment (PXE)](https://github.com/AztecProtocol/aztec-packages/tree/#include_aztec_version/yarn-project/pxe) must have access to the nullifier secret key in order to decrypt the notes.
```

Comment:
General: I'd understood the examples' code needs updating to not make certain assumptions about Note nullifiers, beyond owner references saying nullifier.

Specific to this line's commit, did the owner's decryption keys change too?

Reply:
I think all the examples were updated [here](https://github.com/AztecProtocol/aztec-packages/pull/6405/)

Good catch, changed to ivsk_app